### DISCUSSION TITLE
Starting supportive project in Denmark

### DISCUSSION THREAD

**Discussion by AOvergaard** [globallightingdata/gldf#10] [General]
I would like to announce that a new Danish project with partners from the Technical University of Denmark, Viso Systems, Energy Piano, and Danish Lighting Center will be kicked off tomorrow. 

With the project, we seek to approach the GLDF project from the measurements and standardization side. To make success, we are hoping for fruitful cooperation with DIAL and RELUX  (I already chatted a bit with one engineer about dimming and flicker).
Also, the project leader, Anders Thorseth of DTU, received a nomination for the chairmanship of the technical committee CIE S 025, which prepares one of the most important light measurement standards.

**I assume, that we would like to have a meeting with you in the near future to get the project started on the right foot. who should we contact?**

Some of the objectives of the project are listed here:

**Measurement procedures**
Today, luminaires are typically measured at full load – this applies both in terms of lumen package, energy consumption and light quality. The project seeks to support that new data formats will allow measurements in other states than full loads, which also requires changes to measurement procedures:
It shall be possible for the measurement operator in the software to set up measurement protocols to ensure automatic measurement of all parameters at multiple levels. To this end, the measuring equipment must also be able to communicate with the luminaires (typically digital DALI signal but perhaps also DMX and analogue 1-10V signal) so that the fixtures are dimmed during the measurement process. This communication does not happen today. In particular, the data collected and a number of other luminaire information (part numbers, mechanical functions, sensors, geometry, photos, etc.) could be systematically saved in the new file formats. 
Among the important parameters that are not being tested systematically today are

- Energy efficiency for different light dimming levels
- Power factor and displacement factor for different light dimming levels (this is of prime importance for large lighting systems).
- Standby consumption in smart lighting as well as the ability to turn off or. measure electricity consumption separately for all additional features of the light source, such as the power source. surveillance camera and Wi-Fi booster.
- Information from manufacturers about default color temperature as well as how much efficiency decreases when choosing other color temperatures.
- The relationship between color coordinates, color temperature, raw value and dimming steps
- Flicker in connection with dimming

**Comment by FBremecker** [comment]
Hello AOvergaard,
good to hear that you are starting such an interesting project. I am sure, we will find a way to cooperate in a good way.
For further discussions please contact me from the DIAL side bremecker(at)dialux.com and Robert Heinze from Relux   r.heinze(at)relux.com.

Regards,

Friedrich Bremecker

**Reply by AOvergaard** [reply]
Great - we will do just that. We have just kicked off the project so we're in the planning phase. Nevertheless, I think we would like an online meeting you and Mr. Heinze sometime soon to see how we can help/work together. My company (Viso System) represents the light and power measurements side of things. We also have very good contacts with lighting designers that are essentially the uses of the GLDF. We will contact you soon :-)
